index2.js

```diff
@@ -0,0 +1,13 @@
+arr = []
+arrlen = prompt("enter array length")
+for(var i =0; i<arrlen; ++i){
```

Comment:
Declare variable using `let` instead of `var`. So it can be declared as block scope.